firefox-ios/Client/Assets/Images.xcassets/crossCircleFillExtraLarge.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "crossCircleFillExtraLargeLight.pdf",
+      "filename" : "crossCircleFillExtraLarge 1.pdf",
```

Comment:
can we keep the original filenames ? I think `crossCircleFillExtraLarge(Light|Dark)` are more descriptive.

Reply:
If these images need to be updated then the designers should update them in the Acorn library. We have a script that updates the images automatically.